### PR TITLE
HSC-41: Add missing drug and dosage concepts from HSC

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L15)

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L323)

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L77)
And this is an opportunity to remove [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L78) and [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L266)

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv:434]: Let us consider combining [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L91) , [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L326) , [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L327) and have only [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L294)

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv:439]: Let us use https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L110
Let us also combine [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L240) with [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L271).

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L116)
We combine [these](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L353-L355) as just Insulin

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us combine this to just 'Formula Milk'

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv:442]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L324)

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L130) but probably rename to remove 'sulphate'

File: configuration/concepts/drugs_concepts.csv
Comment (reagan-meant) [inline configuration/concepts/drugs_concepts.csv]: Let us use [this](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/drugs_concepts.csv#L144)

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv:434]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv:439]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv:442]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

File: configuration/concepts/drugs_concepts.csv
Comment (nuhuhmutebi) [reply configuration/concepts/drugs_concepts.csv]: Done

**Comment by rbuisson** [comment]
Thanks.
Let's wait for the release before merging this PR.

**Pull request merged**
